presentations/README.md

```diff
@@ -31,11 +31,11 @@ To use the slides:
     npm install
```

Comment:
The instruction above says 'Navigate to the relevant folder'. This should probably be changed to 'Navigate to the `presentations` folder' so it's clearer where the install should be run.

Reply:
Should be updated now

Reply:
Do they need to run this in the `presentations` folder? If so we should specify this.